use-cases/NVIDIA-GPU-passthrough-and-Kata.md

```diff
@@ -0,0 +1,290 @@
+# Using Nvidia GPU device with Kata Containers
+
+- [Using Nvidia GPU device with Kata Containers](#using-nvidia-gpu-device-with-kata-containers)
+	- [Hardware Requirements](#hardware-requirements)
+	- [Host BIOS Requirements](#host-bios-requirements)
+	- [Host Kernel Requirements](#host-kernel-requirements)
+	- [Install and configure Kata Containers](#install-and-configure-kata-containers)
+	- [Build Kata Containers kernel with GPU support](#build-kata-containers-kernel-with-gpu-support)
+	- [NVIDIA GPU pass-through mode with Kata Containers](#nvidia-gpu-pass-through-mode-with-kata-containers)
+	- [NVIDIA vGPU mode with Kata Containers](#nvidia-vgpu-mode-with-kata-containers)
+	- [Install Nvidia Driver in Kata Containers](#install-nvidia-driver-in-kata-containers)
+	- [References](#references)
+
+
+A Nvidia GPU device can be passed to a Kata Containers container using GPU passthrough (NVIDIA GPU pass-through mode) as well as GPU mediated passthrough (NVIDIA vGPU mode). 
```

Comment:
- s/A Nvidia/An Nvidia/
- Please can you break lines around 70-80 chars to make commenting on them easier (and diffs for future PRs ;)
- s/NVIDIA/Nvidia/

Reply:
updated